plaso/parsers/sqlite_plugins/firefox_downloads.py

```diff
@@ -2,7 +2,7 @@
 """SQLite parser plugin for Mozilla Firefox downloads database files."""
 
 from dfdatetime import posix_time as dfdatetime_posix_time
-
+import json
```

Comment:
style nit: given this module is part of Python the import should come before 3rd party imports